### DISCUSSION TITLE
How to Automatically Update Queue Information in the Web UI ？

### DISCUSSION THREAD

**Discussion by wbadyx** [awslabs/scale-out-computing-on-aws#135] [Q&A]
Hello,

I have modified the cluster_manager/settings/queue_mapping.yml file to customize several queues, including their names and configuration information. How can I automatically update the queue information in the web UI?

Additionally, when I submit tasks to the **job-shared** queue, the queue status always remains  **QUEUED** without any errors. I found that the difference lies in the "scaling mode: multiple_jobs". Do you know what might be causing this issue?
The shell.script:
```
#!/bin/bash
#PBS -N my_job_name
#PBS -V -j oe -o my_job_name.qlog
#PBS -P project_a
#PBS -q job-shared
#PBS -l nodes=1
## END PBS SETTINGS
cd $PBS_O_WORKDIR
cat $PBS_NODEFILE | sort | uniq > mpi_nodes
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/apps/openmpi/5.0.3/lib/
export PATH=$PATH:/apps/openmpi/5.0.3/bin/
# m5.large is 1 cores so -np is 1 * 5 hosts
/apps/openmpi/5.0.3/bin/mpirun --hostfile mpi_nodes -np 1 script.sh > my_output.log
```

Thank you for your help.

**Comment by mcrozes** [comment]
> I have modified the cluster_manager/settings/queue_mapping.yml file to customize several queues, including their names and configuration information. How can I automatically update the queue information in the web UI?

Please note updating `queue_mapping.yml` is only for SOCA and you must also update OpenPBS to reflect your changes. There is a web wrapper for that https://awslabs.github.io/scale-out-computing-on-aws/web-interface/create-your-own-queue/#option1-i-want-to-use-the-same-settings-as-an-existing-queue , otherwise you can run the `qmgr` command directly on the scheduler host (you have the CLI instructions on the same page)

> For your job-shared question, you may need to send us the log directly for troubleshooting. Feel free to email me at my github user a t amazon.com

**Comment by wbadyx** [comment]
Attachment:
![20240724125653](https://github.com/user-attachments/assets/ebfc3af7-b863-4c99-a8c0-1fbc39006d13)
[job-shared.log](https://github.com/user-attachments/files/16357288/job-shared.log)
](url)
![20240724125117](https://github.com/user-attachments/assets/1d95f553-f5d3-4d58-8442-0fabe15b7bcb)
